### ISSUE TITLE
popover() calls after the first on same target with different selectors ignored

### ISSUE THREAD

**Issue by manierim** [twbs/bootstrap#18268]
https://jsfiddle.net/t3zasqrb/

Not sure this is a bug, if not consider it as a request for improvement. 

My use case is that the #container content is dynamically changing, so I need to "delegate" to the #container to manage sets of different popovers (selectors are classes).

**Comment by twbs-lmvtfy** [comment]
Hi @manierim!

You appear to have posted a live example (https://fiddle.jshell.net/DTcHh/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 47, column 5: [W012](https://github.com/twbs/bootlint/wiki/W012): `.navbar`'s first child element should always be either `.container` or `.container-fluid`
* line 16, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.0.0
* line 20, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.0.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by manierim** [comment]
https://jsfiddle.net/t3zasqrb/

**Comment by cvrebert** [comment]
This would seem to require a backward-incompatible or nontrivial fix, so I'll reclassify this as a Bootstrap v4 bug.

**Comment by manierim** [comment]
A little workaround that seems to work correctly with my DOM arrangement: create new "targets" on the fly if the original one is already "occupied":

```javascript
    function popoverize($target,options)
    {
    	if ($target.data('isPopoverTarget'))
    	{
    		$target.wrapInner('<div class="bsPopoverPacthedTarget"></div>');
    		$target = $target.find('.bsPopoverPacthedTarget');

    	}

    	$target.data('isPopoverTarget',true);
    	$target.popover(options);
    }
```

**Comment by mdo** [comment]
This issue is almost five years old—closing as stale. Please open a new issue if the issue still persists.
